src/util/string_utils.cpp

```diff
@@ -79,9 +79,9 @@ void split_string(
   unsigned start=0;
   unsigned i;
 
-  for (i=0; i<n; i++)
+  for(i=0; i<n; i++)
   {
-    if (s[i]==delim)
+    if(s[i]==delim)
```

Comment:
While you are at it: get rid of `unsigned` and use `std::string::size_type` in all those places.